tests/test_project.py

```diff
@@ -1007,40 +1007,42 @@ def op1(job):
             ]:
                 evaluated = 0
                 project.run()
-                self.assertEqual(evaluated, expected_evaluation)
+                assert evaluated == expected_evaluation
 
 
-class BufferedExecutionProjectTest(ExecutionProjectTest):
+class TestBufferedExecutionProject(TestExecutionProject):
 
     def mock_project(self, project_class=None):
-        project = super(BufferedExecutionProjectTest,
+        project = super(TestBufferedExecutionProject,
                         self).mock_project(project_class=project_class)
         project._use_buffered_mode = True
         return project
 
 
-class ExecutionDynamicProjectTest(ExecutionProjectTest):
-    project_class = TestDynamicProject
+class TestExecutionDynamicProject(TestExecutionProject):
+    project_class = DynamicProjectTest
     expected_number_of_steps = 10
 
 
-class BufferedExecutionDynamicProjectTest(BufferedExecutionProjectTest,
-                                          ExecutionDynamicProjectTest):
+class TestBufferedExecutionDynamicProject(TestBufferedExecutionProject,
+                                          TestExecutionDynamicProject):
     pass
 
 
-class ProjectMainInterfaceTest(BaseProjectTest):
-    project_class = TestProject
+class TestProjectMainInterface(TestBaseProject):
+    project_class = ProjectTest
 
     def switch_to_cwd(self):
         os.chdir(self.cwd)
 
-    def setUp(self):
-        super(ProjectMainInterfaceTest, self).setUp()
+    @pytest.fixture(autouse=True)
+    def setUp(self,request):
```

Comment:
```suggestion
    def setup_main_interface(self, request, setUp):
```
Add remove the next line